app/src/main/kotlin/com/hedvig/app/feature/claims/ui/commonclaim/CommonClaimsData.kt

```diff
@@ -36,3 +41,11 @@ data class CommonClaimsData(
     }
   }
 }
+
+fun CommonClaimsData.isFirstVet() = id == "25" || id == "26"
+
+fun getFirstVetIntent(context: Context) = try {
+  context.packageManager.getLaunchIntentForPackage("com.firstvet.firstvet");
+} catch (e: ActivityNotFoundException) {
+  Intent(Intent.ACTION_VIEW, Uri.parse("https://play.google.com/store/apps/details?id=com.firstvet.firstvet"))
```

Comment:
Damn 4.8 stars with 2.77k reviews. They're doing great

Reply:
Yes I saw that as well, really good 👍